app/views/categories/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<div class="container">
+  <h2 class="my-4 headline">Categories</h2>
+  <ul class="list-group">
+    <% @categories.each do |category| %>
+      <li class="list-group-item d-flex justify-content-between category-item align-items-center">
+        <a href="<%= category_transactions_path(category) %>" class="text-decore" style="width: 100%;">
+          <div class="row ">
+            <div class="col-2">
+              <span class="category-icon <%= category.icon %> fa-2x"></span>
+            </div>
+            <div class="col-5 category-name general-text">
+              <%= category.name %>
+            </div>
+            <div class="col-2 text-right category-amount general-text">
+              <%= number_to_currency(category.total_amount) %>
+            </div>
+          </div>
+        </a>
+      </li>
+    <% end %>
+  </ul>
+  <%= link_to "Add", new_category_path, class: 'btn secondary-color my-4' %>
+</div>

```

Comment:
- You have done well so far, but you have not styled this page as it is styled in the design guidelines, please remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

Design template:
![sesregwerg](https://github.com/najibullahjafari/BudgetPro/assets/73607512/66f11882-942d-4941-9663-e3942b911d85)


Your design:
- Please ensure that the button is placed at the bottom of the page as it is in the design guidelines
![wwefawef](https://github.com/najibullahjafari/BudgetPro/assets/73607512/8446de52-be18-478e-947d-8952428592d1)